.github/workflows/arm-ubuntu-base.yaml

```diff
@@ -0,0 +1,43 @@
+name: Build and release base ubuntu images (ARM)
+
+on:
+  pull_request:
+    paths:
+      - 'experimental/arm/base-ubuntu/**'
+    branches: [main]
+  push:
+    paths:
+      - 'experimental/arm/base-ubuntu/**'
+    branches: [main]
+
+defaults:
+  run:
+    working-directory: experimental/arm/base-ubuntu
+
+jobs:
+  # build:
+  #   if: github.event_name != 'push'
+  #   runs-on: ubuntu-latest
+  #   permissions:
+  #     contents: read
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - name: Build image
+  #       run: make
+  release:
+    if: github.event_name != 'pull_request'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
note: there is a `checkout@v3`, some of my pet project jobs started to throw notices about it. In these cases it's only a `v2` to `v3` replace.

Reply:
https://github.com/weaveworks-liquidmetal/image-builder/issues/46